third_party/css/github-markdown.css

```diff
@@ -393,6 +393,11 @@
   /* TODO: Investigate if this should be fixed in dartdoc's styles.css: the `pre` rule has precendece over the dark theme there. */
   background-color: inherit;
 }
+.dark-theme .hljs .hljs-deletion,
+.dark-theme .hljs .hljs-addition {
+  /* TODO: Investigate if this should be fixed in dartdoc's styles.css: the dark theme is not applied there. */
```

Comment:
Could we leave an issue number with the TODO? Otherwise I fear it might never happen.

Reply:
Done.

Reply:
I would write like:
```suggestion
  /* TODO(https://github.com/dart-lang/pub-dev/issues/7101): Investigate if this should be fixed in dartdoc's styles.css: the dark theme is not applied there. */
```